frontend/layouts/noScrollable.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
Ajajaj, we are overcomplicating it! Now you have duplicate code, to fix it you probably have to create component for layouts and I don't like that. We don't have that much centred websites, only login/register + error (index will be scrollable). Maybe just remove this layout, and in those 3 pages manuallly add `max-h-90vh`?